src/site/stages/build/process-cms-exports/schemas/transformed/media-document.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
This is the same as in the `media-document` branch, right? You branched off that branch to get this branch? (How many times do you suppose I can say "branch" in one branch's review? :trollface: Starting to sound like a fake word.)

Reply:
Right - now that that is merged it's time for a rebase :crossed_fingers: